docs/building-ami.md

```diff
@@ -28,17 +28,28 @@ region = eu-west-1
 in `~/.aws/credentials`
 
 ```
-[profile nhsd-website-nonprod]
+[nhsd-website-nonprod]
 aws_access_key_id = ABCAJBCFVV63UJ9X8Y7Z
 aws_secret_access_key = aDghW512HJ...
 mfa_serial = arn:aws:iam::ACCOUNT-ID:mfa/firstname.lastname
 
-[profile nhsd-website-prod]
+[nhsd-website-prod]
 aws_access_key_id = V6BCF3UABCAJVJ9X8Y7Z
 aws_secret_access_key = hjQgau3yAHY5...
 mfa_serial = arn:aws:iam::ACCOUNT-ID:mfa/firstname.lastname
 ```
 
+This can be verified using command `.venv/bin/aws configure list --profile nhsd-website-nonprod`
+
+Output of the above should look e.g. as follows:
+```
```

Comment:
empty line before "```" block

Reply:
Have removed line, is that what you were suggesting ?  Also Travis build fails: "ERROR! the role 'sansible.ntp' was not found ..." Does that need adding, e.g. to main.yml ?

Reply:
sorry - you should have empty line before each "```" (code) block.

I've seen the error - long story short Ansible Galaxy transformed, and you now have to use full "semver" version names. Simply update `ansible/roles/vendor.yml` and put full versions, for NTP it would have to be v1.0.3

Or if you want, wait a bit and I will update it in this branch later today